### DISCUSSION TITLE
Suggestion for an alternative DAO/ecosystem funding.

### DISCUSSION THREAD

**Discussion by r3freshf5** [iota-community/Shimmer-Governance#5] [Ideas]
I have an alternative suggestion for creating a DAO fund, but let me start with something else.

**Why I believe the current approach is a mistake:**
- The IF is finally starting to regain confidence inside and outside the community for being a serious crypto project. We market ourselves as being exactly that and often distinguish ourselves from other projects by not focussing on hype and baseless buzzwords. I don't believe ticking off all the buzzwords in our position is a good idea.
- Selling NFT's without predefining what it is that you get in return is something the IF should not be involved in. Similar to how a roulette game doesn't seem like a proper way to fund the DAO. If you want to sell something, then sell something concrete.
- By only getting 10K people invested in the shimmer network you miss an opportunity to get everyone involved.
- You miss an opportunity to create a respectable story to tell to partners and the rest of the outside world. We're always proud to tell that the iota development was basically crowdfunded. Lets think of a way to generate funds and still have a respectable story to tell.
- Every AMA the IF talks about empowering the community, this is a perfect chance to do it. Why didn't you opt to let the community vote on several DAO funding methods?
- I think people will buy the NFT's so that's fine. I do wonder why though. If you buy 3000 dollars worth of iota tokens and then you stake them to get shimmer tokens as well then you have  immediate value (the iota token) plus a potential future reward (shimmer tokens), plus zero risk because even if shimmer remain worthless you still have your iota tokens. The staking boost must make up for all of that, but we don't even know what that staking boost is.

So now lets get to the point...

**The goal**
The idea is that we want to give the shimmer ecosystem a kickstart by providing a way to fund new projects within that ecosystem. That fund has to come from somewhere. In the current proposal 10000 people who pay for the nft's will provide those funds while everybody else piggybacks on the investment of those 10000 people who take the risk. Which in principle is fine to me, but I think it doesn't fit well with the image the IF has worked so hard for to regain the last few years.  In one line I would define the goal:

`To generate a fund to kickstart the shimmer ecosystem in a way that fits well with the image and identity of iota`

**My proposal**
Rather than only involving 10K people in the creation of the ecosystem, how about we involve everyone. Here's the idea:

- We still have a 90 day staking period to determine the share of shimmer tokens people will get.
- After the 90 day staking period the people who staked their tokens AND made a small one time fixed donation to the DAO funds will get shimmer tokens in the new shimmer network.
- The donation must be made in main net iota tokens.
- For the people who staked but didn't donate, no shimmer tokens are created in the genesis transaction.
- The share of of tokens people will get is still based on the amount/time they staked, but the absolute amount per person is based on their share AND the total initial supply that we want to have in the network.
- The required "donation" makes all those people invested in the network rather than just 10K people who bought an nft.
- The incentive for people to make the donation is that they get shimmer tokens.
- The premise is that shimmer tokens will have value. If we don't believe that premise, than why would people pay 3000 dollar for an nft in the first place?
- Dom replied to this proposal with "isn't that just a crowdfunding?". Yes it is, its a crowdfunding with the benefits of getting shimmer tokens. Whats wrong with that?
- Now to get the same  15 million funding. You need 150K people to donate 100 dollar.
- You can create additional funding by selling the NFT, jerseys and the DAO membership seperately.
- By doing it this way you create a good story to tell to partners and the outside world. The ecosystem is funded by the whole community.
- Yes I know theoretically it isn't a donation because you get something in return, but lets call it donations because that better describes the goal of it: fund the ecosystem together.

Selling staking benefits is always an option later on if its really needed, but why does it need to be the starting point? And why isn't that something the community votes on BEFORE the NFT's are sold?

**Comment by emeliebob** [comment]
Awesome idea!

**Comment by noahzuckerman** [comment]
In theory this is good but A) A forced donation to unlock funds isn't really a donation and B) I think you'd be surprised that majority of people would donate the minimum of lets say 1Mi and much less funds would be raised though this mechanism.

**Reply by r3freshf5** [reply]
No no, the required donation is fixed at a certain price. I suggested 100 dollars (in iota tokens) in my post but it could be anything. You are right its not a true donation its actually a crowdfund with some benefits. But in the end its about funding the ecosystem together.

**Comment by Winmik** [comment]
The goal of the NFT sale is to introduce new money into the ecosystem through new channels and associating perks with those NFTs. Does "selling" IOTA tokens for extra Shimmer tokens accomplish that goal?
I could be wrong, but would argue it doesn't.

**Reply by r3freshf5** [reply]
What happens is that the donations are coming from the iota community (ie. funds they've often been hodling since 2017), those funds were not used for anything productive and now they end up in a community ecosystem fund where it is used for something productive. Generating value for the shimmer network and with that also the  iota network.

And you say part of the goal is to attach perks to the NFT. Why is that the goal? Isn't the goal to fund the DAO, which my suggetion achieves as well, right?

**Reply by 8bn-io** [reply]
Every IOTA sold needs to bought prior. 
👉 By attracting external Investors who want to participate in the NFT auction, offering Genesis NFTs only in exchange for IOTAs is fine IMHO. Many first time IOTA buyers will be rather positive for adoption.

**Comment by Ryan-Crow** [comment]
IF can create three things:

1) Genesis NFTs that provide a fixed income of SMR like a bond (gold silver and Bronze provide different coupons) sold to raise Genesis DAO funds,

2) A pool of SMR that is sufficient to pay the Genesis NFT fixed income from staking rewards indefinitely and is eternally locked. This pool can be created from the IF staked Iota for the SMR genesis,

3) A Genesis Ecosystem DAO which has membership granted by Genesis NFTs above plus the shirt NFTs and this DAO controls the funds raised from the NFT sale and decides how they're used.


It would not only raise capital but demonstrate how fixed income could work on a DLT, along with a good purpose to join the Genesis DAO.
